.github/workflows/autopr_merged.yaml

```diff
@@ -0,0 +1,90 @@
+name: "Tag and release to production"
+
+on: 
+  pull_request:
+    types: [closed]
+
+jobs:
+  tag-release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.base.ref == 'main' && github.event.pull_request.merged && (contains(github.event.pull_request.title, '[AUTO-PR]') || contains(github.event.pull_request.title, '[MANIFEST]')) }}
+    
+    steps:
+      - name: Obtain a Notify PR Bot GitHub App Installation Access Token
+        run: |
+          TOKEN="$(npx obtain-github-app-installation-access-token@1.1.0 ci ${{ secrets.GH_APP_CREDENTIALS_TOKEN }})"
+          echo "::add-mask::$TOKEN"
+          echo "GITHUB_TOKEN=$TOKEN" >> $GITHUB_ENV
+
+      - name: Bump version and push tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.0
+        with:
+          github_token: ${{ env.GITHUB_TOKEN }}
+          release_branches: main
+
+      - name: Checkout main branch
+        uses: actions/checkout@v2
+
+      - name: Get current Prod Version
+        run: |
+          CURRENT_VERSION=`cat VERSION`
+          echo "CURRENT_VERSION=$CURRENT_VERSION" >> $GITHUB_ENV
+
+      - name: Branch protection OFF
```

Comment:
Does disabling the branch protection only apply within the execution scope of this workflow action or to all of the repository?

Reply:
only applies to this workflow and would be `OFF` for a very short duration before being enabled again.